src/main/java/mate/academy/Main.java

```diff
@@ -2,17 +2,22 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import javax.naming.AuthenticationException;
```

Comment:
As already told in README.md
Create your own checked RegistrationException and AuthenticationException to use in register() and login() methods respectively